### ISSUE TITLE
Scraping single games doesn't save to gameslist.xml

### ISSUE THREAD

**Issue by wraybowling** [RetroPie/EmulationStation#27]
It's taken me a while to figure out what's happening... but essentially as soon as I started to get really selective about which games I was adding and scraping I started noticing that the metadata wouldn't stick. It drove me bonkers because scraping a single game seemed to work, but then after rebooting all the titles I'd put box art on would be blank again

**Comment by joolswills** [comment]
metadata wasn't saved on rebooting before. It is now with the latest ES with RetroPie 3.4

Are you using RetroPie 3.4 ?

**Comment by HerbFargus** [comment]
It's also in the documentation: https://github.com/retropie/RetroPie-Setup/wiki/scraper#scraper-not-saving-manual-edits

**Comment by wraybowling** [comment]
Oh. I'm sorry, I thought that by way of running updates I had the latest version. Apparently I don't.

But anyway, I still don't think that saving on reboot was what I was after. I expected that pressing the "save" button would write to the gameslist.xml — especially as a new user to the Raspberry Pi, it can be surprising when it decides to lose power because of a USB device drawing too much voltage for example, or a game crashing. So I don't think that saving on reboot is really that much more beneficial.

**Comment by joolswills** [comment]
How did you update?

We can leave this open to perhaps look at that problem in the future - you should also report it to the main ES project (even though it's not currently developed), so it is referenced there.

**Comment by wraybowling** [comment]
I just was updating retropie-setup via the bash script. I'm sure there's a better place to discuss how i can get up to 3.4 without having to reflash my sd card. Thanks for the information @joolswills

**Comment by joolswills** [comment]
You can update by following this guide - https://github.com/retropie/RetroPie-Setup/wiki/Updating-RetroPie

Thanks.

**Comment by HaloFour** [comment]
I'm having a similar issue with RetroPie 4.1.  I scraped most of the games using the bulk Scraper option in the GUI and that worked fine.  But anytime I attempt to manually scrape individual games the updates are lost anytime I restart EmulationStation.

I can see from the logs that the gamelist at `/home/pi/.emulationstation/gamelists` is being parsed.  If I modify those gamelists manually I do see those changes when EmulationStation is restarted.  However, no changes made within EmulationStation seem to be getting saved to these files.  That includes rescraping existing ROMs that have metadata or scraping ROMs that don't yet have a node in the gamelist.  In either case the updates never make it back into the `gamelist.xml`.  I have tried manually quitting/restarting EmulationStation from the UI and the files are not updated.  There are no errors in `/home/pi/.emulationstation/es_log.txt`.
